src/osemgrep/cli_login/Logout_subcommand.ml

```diff
@@ -18,7 +18,9 @@ let run (conf : Login_CLI.conf) : Exit_code.t =
   let settings = Semgrep_settings.load () in
   let settings = Semgrep_settings.{ settings with api_token = None } in
   if Semgrep_settings.save settings then (
```

Comment:
Maybe print a message if someone is already logged out?

Reply:
added
```
 WARN  You are not logged in! This command had no effect.
 ```